docs/user/reporting/index.asciidoc

```diff
@@ -7,37 +7,38 @@
 :frontmatter-description: {kib} provides you with several options to share *Discover* saved searches, dashboards, *Visualize Library* visualizations, and *Canvas* workpads with others, or on a website.
 :frontmatter-tags-products: [kibana]
 
-{kib} provides you with several options to share *Discover* saved searches, dashboards, *Visualize Library* visualizations, and *Canvas* workpads.
+{kib} provides you with several options to share *Discover* saved searches, dashboards, *Visualize Library* visualizations, and *Canvas* workpads. These sharing options are available from the *Share* menu in the toolbar.
 
-You access the options from the *Share* menu in the toolbar. The sharing options include the following:
+[float]
+[[share-a-direct-link]]
+== Share with a direct link
 
-* *PDF Reports* &mdash; Generate and download PDF files of dashboards, visualizations, and *Canvas* workpads. PDF reports are a link:https://www.elastic.co/subscriptions[subscription feature].
+You can share direct links to saved searches, dashboards, and visualizations. When clicking **Share**, look for the **Links** tab to get the shareable link and copy it. When sharing an object with unsaved changes, you're getting a temporary link that might break in the future for example in case of upgrade. Save the object to get a permanent link instead.
```

Comment:
```suggestion
You can share direct links to saved searches, dashboards, and visualizations. When clicking **Share**, look for the **Links** tab to get the shareable link and copy it. 

TIP: When sharing an object with unsaved changes, you get a temporary link that might break in the future, for example in case of upgrade. Save the object to get a permanent link instead.
```